share/revng/test/tests/analysis/DetectStackSize/dynamic_native/memcpy-from-stack-arguments.c.model.yml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
I'm putting this here because I don't know where else to put it.

Looking at my comment above on using `slt` vs `sle`, if it turns out I am right and it was a bug, I'd like to have a test for that, so that whenever we have to change this it doesn't slip back in.